### PR TITLE
Documentation : Aside

### PR DESCRIPTION
**Pull request by NehaChandra22**
Created Documentaion-page and added Aside feature in it.
image
Responsive -
image
image

### CODE REVIEW

File: documentation.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      rel="shortcut icon"
+      href="./assets/Images/favvv.png"
+      type="image/x-icon"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css"
+      integrity="sha512-DTOQO9RWCH3ppGqcWaEA1BIZOC6xxalwEsw9c2QQeAIftl+Vegovlnee1c9QX4TctnWMn13TZye+giMm8e2LwA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
```
Comment (Asuraking1n) [inline documentation.html:16]: make sure you know what these attribute means

File: assets/Js/script.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+const sidebar = document.querySelector('.left-sidebar');
+const hidehamburgerL = document.querySelector('#hamburger-L');
```
Comment (Asuraking1n) [inline assets/Js/script.js:2]: use camel case naming convention in javascript

File: assets/Js/script.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+const hidehamburgerR = document.querySelector('#hamburger-R');
+const hidecrossL = document.querySelector('#cross-icon-L');
+const hidecrossR = document.querySelector('#cross-icon-R');
+const navbar = document.querySelector('.navbar');
+const Darkmode = document.querySelector('.dark-mode');
```
Comment (Asuraking1n) [inline assets/Js/script.js:7]: here as well rename it as darkMode

File: assets/styles/cui-use.css [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+.menu li {
+  list-style: none;
+  margin: 15px 0px 15px 20px;
+  font-size: 1.1rem;
+  border-radius: 5px;
+  padding: 4px;
+  color: #a4629d;
+}
+
+.menu-2 .box-icon {
```
Comment (Asuraking1n) [inline assets/styles/cui-use.css:66]: don't give name like menu-2. give something meaningfull

File: assets/styles/style.css [hunk 5]
```diff
@@ -0,0 +1,697 @@
+@import url(./variable.css);
+@import url(.component-library.css);
+@import url("https://fonts.googleapis.com/css2?family=Dosis&family=Genos:wght@700&family=Poppins&family=Varela+Round&display=swap");
```
Comment (Asuraking1n) [inline assets/styles/style.css:3]: Make sure to put these fonts on top, for better formating

**Review by Asuraking1n — CHANGES_REQUESTED** [review]

File: documentation.html (hunk 1 above)
Comment (NehaChandra22) [reply documentation.html:16]: just copied the CDN link, I'll look into these.

File: assets/Js/script.js (hunk 3 above)
Comment (NehaChandra22) [reply assets/Js/script.js:7]: it is in red color bcz its not in camel case?

**Comment by shuklasmriti** [comment]
you should also link your git icon with your github link in the navbar.

**Pull request merged**